content/freeze.md

```diff
@@ -55,7 +56,9 @@ No Freeze
 The **No Freeze** feature is a setting on an account that permanently gives up the ability to freeze counterparties. A business can use this feature to treat its issued funds as "more like physical money" in the sense that the business cannot interfere with customers trading it among themselves. The NoFreeze setting has two effects:
 
 * The issuing account can no longer use enable Individual Freeze on any counterparty.
+    [//]: # ("can no longer enable")
 * The issuing account can still enable Global Freeze to enact a global freeze, but the account cannot _disable_ Global Freeze.
+    [//]: # (To clarify, perhaps: "the account cannot disable Global Freeze once it's enabled")
```

Comment:
OK. I see that the logic here is "Once you enable this setting, the following things are true," which covers my concern. Comment retracted.